readarr/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: readarr_server_1
+      APP_PORT: 8787
+      PROXY_AUTH_WHITELIST: "/api/*"
+
+  server:
+    image: linuxserver/readarr:0.2.4-develop
```

Comment:
Hey @Choff3 , It's advised that you use an Image with it's `image digest`. Checkout this example here:

https://github.com/getumbrel/umbrel-apps/blob/master/btc-rpc-explorer/docker-compose.yml#L10

This is necessary because the developer can update the application to the same tag, and when umbrel restarts it will pull the changes which aren't verified. Adding a digest solves that issue. 

you can get the Digest by doing: `docker pull linuxserver/readarr:0.2.4-develop`. It will print the Digest.

```
Digest: sha256:5a6d8c07a93efc5d6b80479354f3dfac22d69b79c2354aa1d8da7de3982d4060
```